website/source/layouts/digitalocean.erb

```diff
@@ -33,6 +33,10 @@
           <a href="/docs/providers/do/r/droplet.html">digitalocean_droplet</a>
                     </li>
 
+                  <li<%= sidebar_current("docs-do-resource-firewall") %>>
+                    <a href="/docs/providers/do/r/firewall.html">digitalocean_firewall</a>
+                  </li>
+
                   <li<%= sidebar_current("docs-do-resource-floating-ip") %>>
```

Comment:
digitalocean.erb side has been taken care of.  Thanks @stack72, for your guidance!  Really appreciated!